cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
```

Comment:
In the `validate` method, ensure that `movie_session` is present in `attrs` before accessing it to avoid potential `KeyError`. Consider using `attrs.get('movie_session')` and handling the case where it might be `None`.